src/main/kotlin/no/nav/familie/klage/ekstern/EksternBehandlingController.kt

```diff
@@ -61,6 +61,12 @@ class EksternBehandlingController(
 
     @PostMapping("/opprett")
     fun opprettBehandling(@RequestBody opprettKlageBehandlingDto: OpprettKlagebehandlingRequest) {
+        // TODO burde man sende med FagsystemVedtak i requesten i stedet for eksternBehandlingId ?
+        //  Hvis ikke så må klage hente FagsystemVedtak fra ef-sak i det att eks ef-sak gjør et kall till klage
+        // [eksternBehandlingId] funksjonaliteten er ikke tatt i bruk ennå fra ef-sak
```

Comment:
Kanskje, evt ikke godta noe som helst inntil videre? For da får vi ikke mer "dirty" data iallefall?

Reply:
Får vel ikke valgt dette i EF sak i dag?

Reply:
👍 Fjerner fra kontrakter